powerquery-docs/connectors/celonis-ems.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Power Query Celonis EMS connector
+description: Provides basic information, prerequisites, and connection steps for Celonis EMS
+author: capono
+
+ms.date: 02/17/2023 
+ms.author: 
+LocalizationGroup: reference
+---
+
+# Celonis EMS
+
+  
+
+## Summary
+
+| Item | Description |
+| ---- | ----------- |
+| Release State | General Availability |
+| Products | Power BI (Datasets) |
```

Comment:
This also needs to include Power BI (Dataflows)

Reply:
Power BI (Dataflows) was added